src/scss/vars/_colors.scss

```diff
@@ -13,6 +13,7 @@
     --ons-color-grey-15: #e2e2e3;
     --ons-color-grey-5: #f5f5f6;
     --ons-color-white: #fff;
+    --ons-color-pale-grey: #efeff0;
```

Comment:
Think this should be called `--ons-color-grey-10` and be put in between 15 and 5